src/plugins/websearch/src/extension.cpp

```diff
@@ -253,7 +253,9 @@ std::vector<Websearch::SearchEngine> defaultSearchEngines = {
     {"Ebay",          "eb ",  ":ebay",      "http://www.ebay.com/sch/i.html?_nkw=%s"},
     {"GitHub",        "gh ",  ":github",    "https://github.com/search?utf8=✓&q=%s"},
     {"Wikipedia",     "wiki ",":wikipedia", "https://en.wikipedia.org/w/index.php?search=%s"},
-    {"Wolfram Alpha", "=",    ":wolfram",   "https://www.wolframalpha.com/input/?i=%s"}
+    {"Wolfram Alpha", "=",    ":wolfram",   "https://www.wolframalpha.com/input/?i=%s"},
+    {"DuckDuckGo",    "dd",   ":duckduckgo","https://duckduckgo.com/?q=%s"},
+    {"Torrentz2",     "tz",   ":torrentz2", "https://torrentz2.eu/search?f=%s"}
```

Comment:
The intention of this site smells fishy. I can't pull this. Remove it and ill merge